client/components/sidebar/iaasSection/cloud-sidebar/project-sidebar.service.js

```diff
@@ -69,7 +69,7 @@ class CloudProjectSidebar {
         this.addSearchKeys(subSection);
         this.SidebarMenu.addMenuItem({
             title: this.$translate.instant("cloud_sidebar_pci_infrastructure"),
-            icon: "infrastructure",
+            // icon: "infrastructure",
```

Comment:
Why is it in comment?

Reply:
it should not I have forgotten to delete it.
Done.